modules/vantagecloud-lake/pages/vantagecloud-lake-compute-cluster-airflow.adoc

```diff
@@ -0,0 +1,341 @@
+= Manage Vantage Cloud Lake Compute Cluster with Airflow
+:experimental:
+:page-author: Satish Chinthanippu
+:page-email: satish.chinthanippu@teradata.com
+:page-revdate: July 11th, 2024
+:description: Manage Vantage Cloud Lake Compute Cluster with Airflow
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, airflow, workflow, teradatasql, ipython-sql, cloud computing, machine learning, vantagecloud, vantagecloud lake, lake
+:dir: vantagecloud-lake-compute-cluster-airflow
+:auxdir: vantagecloud-lake-compute-cluster-airflow
+
+== Overview
+
+This tutorial demonstrates the process of utilizing the https://airflow.apache.org/docs/apache-airflow-providers-teradata/stable/operators/index.html[Teradata Airflow Compute Cluster Operators] to manage the Vantage Cloud Lake Compute Cluster. The objective is to execute dbt models on the Lake Compute Cluster.
+
+== Prerequisites
+* Access to a VantageCloud Lake environment
```

Comment:
Should we say "Ensure you have the necessary credentials and access rights to use Teradata Vantage Lake."

Reply:
Modified as Ensure you have the necessary credentials and access rights to use Teradata Vantage Lake.